lib/commons/rscommons/prism.py

```diff
@@ -0,0 +1,162 @@
+import csv
+import statistics
+from osgeo import ogr
+from shapely.wkt import loads as wkt_load
+from shapely.geometry import mapping, shape, Polygon, MultiPolygon
+from shapely.ops import unary_union
+from rscommons.shapefile import get_transform_from_epsg
+
+def load_mean_annual_data(gpkg, input_csv, attribute_name):
+    '''Loads PRISM attribute csv file into PRISM geopackage as mean annual value'''
```

Comment:
Really small thing but can we get automatic code comments on all the functions?

In VSCode if you type """ and then hit enter it usually fills it in for you.

If that doesn't work open up the workspace setting in VSCode and make sure `docstringFormat` is set to `google`.

```
"autoDocstring.docstringFormat": "google"
"autoDocstring.quoteStyle": "\"\"\""
```

So you should get something like:

```python
def buffer_by_field(flowlines, field, epsg, conversion=1):
    """[summary]

    Args:
        flowlines ([type]): [description]
        field ([type]): [description]
        epsg ([type]): [description]
        conversion (int, optional): [description]. Defaults to 1.

    Returns:
        [type]: [description]
    """

    driver = ogr.GetDriverByName("ESRI Shapefile")
```

Reply:
Done for prism.py, will continue to do so in future.